pkgs/development/compilers/ponyc/default.nix

```diff
@@ -1,85 +1,105 @@
-{ stdenv, fetchFromGitHub, llvm, makeWrapper, pcre2, coreutils, which, libressl, libxml2,
-  cc ? stdenv.cc, lto ? !stdenv.isDarwin }:
+{ clangStdenv, fetchFromGitHub, fetchurl, llvm, makeWrapper, pcre2, coreutils, which, libressl, libxml2, cmake, z3,
```

Comment:
Could you explain why do you need clangStdenv?

Reply:
Sure, ponyc is a compiler for the language ponylang (ponylang.io) and they've dropped support for gcc in favor of clang.

![image](https://user-images.githubusercontent.com/816465/94355899-47fecb00-0056-11eb-8e39-32351a955e68.png)

Reply:
I (personally) don't have the requisite C skills to fix any clang/gcc specific issues.